backend/inspirehep/editor/utils.py

```diff
@@ -0,0 +1,345 @@
+import re
```

Comment:
call this `authorlist.py`? it's not generic utils, so the file name is a bit of a misnomer.